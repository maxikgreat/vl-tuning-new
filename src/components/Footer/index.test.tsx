import {render} from '@testing-library/react';

import {Footer} from './index';

describe('Footer component', () => {
	it('should render correctly and match snapshot', () => {
		const {asFragment, container} = render(<Footer />);

		expect(container.querySelector('footer')).toBeInTheDocument();
		expect(asFragment).toMatchSnapshot();
	});

	it('should return null', () => {
		const {container} = render(<Footer isVisible={false} />);

		expect(container.querySelector('footer')).toBeFalsy();
	});
});
