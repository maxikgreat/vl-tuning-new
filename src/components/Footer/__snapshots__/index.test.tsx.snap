// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render correctly and match snapshot 1`] = `[Function]`;
